i2c/lcd_1602_i2c/i2c_lcd.h

```diff
@@ -0,0 +1,26 @@
+/**
+* Copyright (c) 2025 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef I2C_LCD_H
+#define I2C_LCD_H
+#include <stdbool.h>
+#include <hardware/i2c.h>
+
+typedef struct i2c_lcd* i2c_lcd_handle;
+
+i2c_lcd_handle i2c_lcd_init(i2c_inst_t* handle, uint8_t address);
```

Comment:
If you're intending this to be a "library-like example", then it's probably worth adding some comments to this header-file, with a  brief explanation of what each function does, and what arguments it expects?

Reply:
Fixed